adal/src/main/java/com/microsoft/aad/adal/Logger.java

```diff
@@ -234,6 +287,77 @@ private void log(String tag, String message, String additionalMessage, LogLevel
         }
     }
 
+    private void commonCoreWrapper(String tag, String message, String additionalMessage, LogLevel logLevel,
```

Comment:
lets talk about this tomorrow, i am confused why ADAL is calling into the common core